src/Stratis.Bitcoin/P2P/Peer/NetworkPeer.cs

```diff
@@ -276,8 +276,8 @@ public bool IsConnected
             this.loggerFactory = loggerFactory;
             this.dateTimeProvider = dateTimeProvider;
 
-            this.preferredTransactionOptions = network.NetworkOptions;
-            this.SupportedTransactionOptions = network.NetworkOptions & ~NetworkOptions.All;
+            this.preferredTransactionOptions = parameters.PreferredTransactionOptions;
```

Comment:
Can you please explain this? I don't understand how the NetworkOptions become the options of another Peer. Is it because now that the NetworkOptions in Network are gone, that's the only way you have to know what the options are?